WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -253,6 +252,14 @@ protected void onCreate(Bundle savedInstanceState) {
         mShowAztecEditor = AppPrefs.isAztecEditorEnabled();
         mShowNewEditor = AppPrefs.isVisualEditorEnabled();
 
+        if (mShowAztecEditor) {
```

Comment:
Rather than have different layouts for each editor, can we simply stick with one and set the desired margin in code?